ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -27,22 +27,29 @@
 
               <!-- MHR Information Review Section -->
               <template v-if="isReviewMode">
-                <HomeOwnersTable
-                  class="px-7"
-                  isMhrTransfer
-                  isReadonlyTable
-                  :homeOwners="reviewOwners"
-                  :currentHomeOwners="getMhrTransferCurrentHomeOwners"
-                />
+                <section>
+                  <HomeOwnersTable
+                    class="px-7"
+                    isMhrTransfer
+                    isReadonlyTable
+                    :homeOwners="reviewOwners"
+                    :currentHomeOwners="getMhrTransferCurrentHomeOwners"
+                  />
+                </section>
+                <section id="transfer-certify-section" class="mt-10 pt-4 pb-60px">
+                  <CertifyInformation
+                  :setShowErrors="validateAuthorizationError"
+                  @certifyValid="authorizationValid = $event"
+                  />
+                </section>
               </template>
 
               <!-- MHR Information Section -->
               <template v-else>
                 <HomeOwners isMhrTransfer class="mt-n2" />
+                <TransferDetails :validateTransferDetails="validateTransferDetails" />
               </template>
             </section>
-
-            <TransferDetails :validateTransferDetails="validateTransferDetails" />
```

Comment:
Besides the Review, TransferDetails should also be showing here, but be dynamic.